.github/workflows/test.yml

```diff
@@ -0,0 +1,54 @@
+name: test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Install Python test dependencies in a venv
+        run: |
+          python3 -m venv venv
+          . venv/bin/activate
+          pip install -q -U pip
+          pip install --progress-bar=off pytest
+
+      - name: Install diff-pdf
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends \
+            libpoppler-glib-dev \
+            libwxgtk3.0-gtk3-dev
+
+          git clone --depth=1 --branch master https://github.com/vslavik/diff-pdf.git
```

Comment:
Is it not available on Ubuntu `apt-get`?

Reply:
Hmmm... It seems that it is not. It seems that it is developed by Fedora loving Debian hating czech.

Reply:
Can it check out not the latest master but some concrete tag (`v0.5.1`)?

Though I am not sure if that would be more secure and you can't override tags if some malicious actor would take control of `diff-pdf` repo.

Reply:
@petraszd good idea. pinned to v0.5.1